src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -9,55 +9,106 @@ The browser module APIs aim for rough compatibility with the [Playwright API for
 
 Note that because k6 does not run in NodeJS, the browser module APIs will slightly differ from their Playwright counterparts.
 
+You can find examples of using [the browser module API](#browser-module-api) in our [getting started guide](/using-k6-browser).
+
 <Blockquote mod="note" title="">
 
-To work with the browser module, make sure you are using [k6 version 0.43.0](https://github.com/grafana/k6/releases/tag/v0.43.0) or above.
+To work with the browser module, make sure you are using the latest [k6 version](https://github.com/grafana/k6/releases).
```

Comment:
Do you think it'd make sense to include a callout here that there were breaking changes to this module on version 0.46? So if people are coming to this page and wondering where `chromium` is, they have some guidance on where to go?

Reply:
We discussed this with @mdcruz and @ppcano and have decided to leave the exact version in the documentation. WDYT @mdcruz and @ppcano?